### DISCUSSION TITLE
Random delete delay (bot detection evasion)

### DISCUSSION THREAD

**Discussion by SinanAkkoyun** [victornpb/undiscord#342] [3. Ideas]
Hi! It would be really nice if randomized delays get implemented!

**Comment by ghost** [comment]
Why is this not a thing, this would erase the case of a ban

**Comment by gohamstergo** [comment]
not sure if bumps work. but bump.

**Reply by victornpb** [reply]
I have something in the works, which can be better than slapping randomized delays.

> Why is this not a thing, this would erase the case of a ban

I had plans for this since 1.0, but this is kinda of a cat and mouse race, if I added this early they would throttle everyone more aggressive and it wouldn't be effective anymore at this point. So the reason is you keep things as simple as possible until you have no choice, because you don't want to rise the difficulty of that game too fast. I hope that makes sense
